### PR TITLE
feat(vd): set common logger slog for controller

### PR DESCRIPTION
**Pull request by Isteb4k**
## Description

Refactored vd controller to common logger interface (slog)

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [X] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go [hunk 1]
```diff
@@ -316,7 +316,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		r.logger.Debug("Run handler", slog.String("name", h.Name()))
 		res, err := h.Handle(ctx, s)
 		if err != nil {
-			r.logger.Error("The handler failed with an error", slog.String("name", h.Name()), log.SlogErr(err))
+			r.logger.Error("The handler failed with an error", slog.String("name", h.Name()), logger.SlogErr(err))
```
Comment (yaroslavborbat) [inline images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go:319]: Use logger.SlogHandler

File: images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go [hunk 2]
```diff
@@ -316,7 +316,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		r.logger.Debug("Run handler", slog.String("name", h.Name()))
```
Comment (yaroslavborbat) [inline images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go:316]: Use logger.SlogHandler

**Review by yaroslavborbat — DISMISSED** [review]

File: images/virtualization-artifact/pkg/controller/vd/vd_reconciler.go [hunk 3]
```diff
@@ -71,19 +71,19 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, nil
 	}
 
-	r.logger.Info("Start reconcile VD", slog.String("namespacedName", req.String()))
+	log.Info("Start vd reconciliation")
```
Comment (LopatinDmitr) [inline images/virtualization-artifact/pkg/controller/vd/vd_reconciler.go:74]: I would put it in DEBUG

File: images/virtualization-artifact/pkg/controller/cvi/cvi_controller.go
Comment (LopatinDmitr) [inline images/virtualization-artifact/pkg/controller/cvi/cvi_controller.go:1]: `LogConstructor: logger.NewConstructor(log)` shouldn't this construction be used here?

File: images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go (hunk 1 above)
Comment (Isteb4k) [reply images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go:319]: This pr not about vm, only vd

File: images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go (hunk 2 above)
Comment (Isteb4k) [reply images/virtualization-artifact/pkg/controller/vm/vm_reconciler.go:316]: PR about vd, not vm

File: images/virtualization-artifact/pkg/controller/cvi/cvi_controller.go
Comment (Isteb4k) [reply images/virtualization-artifact/pkg/controller/cvi/cvi_controller.go:1]: Whis PR only about vd, not about cvi

**Review by yaroslavborbat — APPROVED** [review]

**Review by LopatinDmitr — APPROVED** [review]

**Pull request merged**